client/signup/navigation-link/index.jsx

```diff
@@ -112,7 +112,8 @@ export class NavigationLink extends Component {
 		if ( this.props.direction === 'forward' ) {
 			this.props.submitSignupStep(
 				{ stepName: this.props.stepName },
-				this.props.defaultDependencies
+				this.props.defaultDependencies,
+				true
```

Comment:
When the direction equals _forward_, it means that we deal with a "skip event", hence `isSkipped` is set to `true`